kamelets/aws-ddb-sink.kamelet.yaml

```diff
@@ -107,17 +113,24 @@ spec:
   - "camel:aws2-ddb"
   - "camel:kamelet"
   template:
+    beans:
+    - name: dataTypeRegistry
```

Comment:
Not sure if this should be there as there will be an instance of the registry for each kamelet instantiation

Reply:
Good point. How can I make sure to add this to the Camel registry as a singleton service?

Reply:
it must be done outside kamelets but eventually you can explore about not needing a registry at all ad lazy loading converters.

Reply:
see https://github.com/apache/camel-k/issues/3845